agents/PCS-Large-Exhange-Rate-Update/src/agent.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { ethers, Finding, FindingSeverity, FindingType, HandleTransaction } from "forta-agent";
+import { provideBotHandler } from "./agent";
+import { encodeParameter } from "forta-agent-tools/lib/utils";
+import { TestTransactionEvent, createAddress, MockEthersProvider } from "forta-agent-tools/lib/tests";
+import { getPancakePairCreate2Address } from "./utils";
+import { ERC20ABI, ER_THRESHOLD, PANCAKE_PAIR_ABI } from "./constants";
```

Comment:
Please avoid last PR reviews issues, Use `ERC20_ABI`, make use of `_`in uppercase constants.

Reply:
Done